python/ray/test_utils.py

```diff
@@ -336,6 +336,25 @@ def dicts_equal(dict1, dict2, abs_tol=1e-4):
     return True
 
 
+def same_elements(elems_a, elems_b):
+    """Checks if two iterables (such as lists) contain the same elements. Elements
```

Comment:
Can you instead check `set(a) == set(b)`?

Reply:
unfortunately can't because dictionaries aren't hashable